src/OrbitGl/TrackContainer.h

```diff
@@ -58,9 +58,12 @@ class TrackContainer final : public CaptureViewElement {
           iterator_timer_info,
       const absl::flat_hash_map<uint64_t, uint64_t>& iterator_id_to_function_id);
   void UpdateVerticalScrollUsingRatio(float ratio);
-  void IncrementVerticalScroll(float delta_y);
   [[nodiscard]] float GetVerticalScrollingOffset() const { return vertical_scrolling_offset_; }
   void SetVerticalScrollingOffset(float value);
+  void OnVerticalScroll(int delta);
```

Comment:
I find it hard to understand what `delta` means here, and what's the difference to `IncrementVerticalScroll`. Maybe this is a bit of a bigger discussion about our naming, and when we use e.g. `OnSomething()`. Could you at least add a comment explaining this?

Reply:
As I wrote before, What do you think if I split OnVerticalScroll to OnVerticalScrollUp and OnVerticalScrollDown and everything is managed here in TrackContainer?

Reply:
You are right, I modified and now everything is managed in CaptureWindow as you suggested. Thanks!